src/components/TimerPlayer/TimerPlayer.js

```diff
@@ -63,20 +66,28 @@ const TimerPlayer = ({ id, timer, deleteTimer }) => {
       interval = setInterval(() => {
         setRemainingTime(remainingTime => remainingTime - 1);
       }, 1000);
+      if (remainingIntervalTime === restTime && soundOn) {
```

Comment:
Nit pick, but you could wrap them both in a `soundOn` check instead of doing it in each. You could also make it an else as i presume you never want them to both play together